CardCollectionView/Source/Controller/View Controller/CardCollectionViewController.swift

```diff
@@ -76,36 +77,38 @@ class CardCollectionViewController: UICollectionViewController {
     
     // MARK: - Status Bar
     
-    override func preferredStatusBarStyle() -> UIStatusBarStyle {
-        return .LightContent
-    }
+//    override func preferredStatusBarStyle() -> UIStatusBarStyle {
```

Comment:
`preferredStatusBarStyle` is now implemented as a property in Swift 3, this should be reflected instead of commenting out the functionality e.g.: ```override var preferredStatusBarStyle: UIStatusBarStyle {
        return .lightContent
    }```